content/en/about-pixie/05-faq.md

```diff
@@ -0,0 +1,235 @@
+
+Yes, Pixie can capture requests with only one endpoint within the cluster. For example, if a service makes a call to an external mySQL database which is not monitored by Pixie, Pixie will still be able to capture the SQL calls. However, for endpoints outside of the cluster, Pixie won’t be able to resolve the remote address to a pod/service name, so you’ll need to know the endpoint’s IP address.
+
+#### Can I capture system metrics like CPU?
+
+Yes, Pixie captures machine-level resource metrics on Linux and other Unix systems such as CPU usage, memory, disk utilization and network bandwidth.
+
+#### What is the performance impact?
+
+Typical performance overhead for node CPU usage is between 2-5%, depending on the amount and type of traffic. We are in the process of optimizing this, however, so watch for that to go down even further.
+
+#### Which languages are supported for continuous profiling?
+
+Go, C++ and Rust are currently supported. The [roadmap](/about-pixie/roadmap/#roadmap) contains plans to expand support to other languages.
+
+## How do I...?
+
+#### How do I export data from the Pixie platform? Import data?
+
+Pixie offers two [client libraries](/reference/api/overview/) (Go, Python) to allow developers to easily integrate Pixie observability data into their existing stack.
+
+Pixie does not currently offer data ingestion.
+
+The [roadmap](/about-pixie/roadmap/#roadmap) contains plans to support exporting or ingesting data in a variety of additional formats.
+
+#### How do I share a Pixie dashboard with others?
+
+*Self-hosted Pixie*
+
+To share data from the same cluster, [send an invitation through the Admin UI](/installing-pixie/quick-start/#invite-others-to-your-organization-(optional)).
+
+*Community Cloud*
+
+To share data from the same cluster, both parties will need to register for Pixie accounts using emails from the same gsuite email domain. For example, if both parties sign up using a @mycompany.com email address, then both parties will automatically be grouped into the same mycompany org and see the same results.
+
+#### How do I set up RBAC?
+
+Pixie does not yet offer full RBAC support. However, you can enable “Approvals” under the [Org Settings](https://work.withpixie.ai/admin/org) tab on the admin page. This requires each user who registers in the org to be manually approved on the [Users](https://work.withpixie.ai/admin/users) tab before they can log in.
+
+#### How do I send alerts?
+
+Pixie does not offer alerting. However, alerts can be set up using Pixie’s API. For example, see the [slackbot alert tutorial](/tutorials/slackbot-alert/).
+
+For comprehensive alerting, we recommend integrating with third-party observability tools (such as New Relic’s integration).
+
+#### How do I delete a cluster?
+
+The UI does not currently support deleting clusters. If you’d like to rename your cluster, you can redeploy Pixie with the `--cluster_name` deploy flag.
```

Comment:
Some of these overlap with our existing pages. Should we link them instead so that we don't have to worry about stale content?